erpnext/healthcare/doctype/inpatient_record/inpatient_record.js

```diff
@@ -147,7 +147,12 @@ let transfer_patient_dialog = function(frm) {
 			if(dialog.get_value('service_unit')){
 				service_unit = dialog.get_value('service_unit');
 			}
-			if(!check_in){
+			if(check_in > frappe.datetime.now_datetime()){
+				frappe.msgprint({
+					title: __('Not Allowed'),
+					message: __('Check-in time should not be later than current time'),
```

Comment:
```suggestion
					message: __('Check-in time cannot be greater than the current time'),
```
To be consistent with the error messages